scripts/establish_conda_env.sh

```diff
@@ -80,7 +80,7 @@ function find_conda_defs ()
       then
         CONDA_DEFS="/c/ProgramData/Miniconda2/etc/profile.d/conda.sh";
       else
-        CONDA_DEFS="$HOME/miniconda2/etc/profile.d/conda.sh";
+        CONDA_DEFS="$HOME/miniconda3/etc/profile.d/conda.sh";
```

Comment:
this shouldn't be changed globally; this is particular to a build. I would recommend reverting this for now; it should be taken care of in a separate merge request when we make python3 the default.

Reply:
Fixed, change back into the original one